pkg/metadata/host/host.go

```diff
@@ -189,7 +209,16 @@ func getNetworkMeta() *NetworkMeta {
 		log.Infof("could not get network metadata: %s", err)
 		return nil
 	}
-	return &NetworkMeta{ID: nid}
+
+	networkMeta := &NetworkMeta{ID: nid}
+
+	publicIPv4, err := getPublicIPv4()
```

Comment:
I think we need an option to disable this. In the past IP was considered a sensitive information for multiple customers. We might want to loop with security or product on this.

WDYT ?

Reply:
@leeavital Thoughts?

Reply:
FWIW we already send the IP as reported by net.Interfaces().   Plenty of existing users expose have their public IPs there (unlike in AWS.)

If we have a flag to disable that I think we should use it to control this too.   Otherwise, this is reproducing existing functionality in AWS.